### PR TITLE
Correct wellKnownTypeConv function references

### LINKED ISSUE #808: Unable to add protobuf wrappers in url template option
## Pre
1. Using string value from: https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/wrappers.proto
2. protoc version: 3.6.1

## Steps
My proto file looks like this:
>[helloworld.proto](https://github.com/mayank-dixit/examples/blob/master/grpc-gateway/protobuf-wrappers-in-url-template/helloworld.proto)
```
syntax = "proto3";

package helloworld;

import "google/api/annotations.proto";
import "google/protobuf/wrappers.proto";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {
    option (google.api.http) = {
      get: "/say/{strVal}"
    };
  }
}

message HelloRequest {
  google.protobuf.StringValue strVal = 5;
}

message HelloReply {
  string message = 1;
}

```

And then I try compile protos:
```
protoc -I/usr/local/include -I. \
  -I$GOPATH/src \
  -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
  --go_out=plugins=grpc:. \
  helloworld/helloworld.proto
```
Works fine. And then I try to generate reverse proxy using:
```
protoc -I/usr/local/include -I. \
  -I$GOPATH/src \
  -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
  --grpc-gateway_out=logtostderr=true:. \
  helloworld/helloworld.proto
```

### expected:
Successful compilation

### result: 
```
--grpc-gateway_out: aggregate type TYPE_MESSAGE in parameter of Greeter.SayHello: strVal
```

### PR DESCRIPTION
**Pull request by johanbrandhorst**
Correct wellKnownTypeConv function references.

Fixes #808

### CODE REVIEW

File: protoc-gen-grpc-gateway/descriptor/types.go [hunk 1]
```diff
@@ -453,14 +453,14 @@ var (
...
-		".google.protobuf.StringValue": "runtime.StringValue",
-		".google.protobuf.BytesValue":  "runtime.BytesValue",
-		".google.protobuf.Int32Value":  "runtime.Int32Value",
-		".google.protobuf.UInt32Value": "runtime.UInt32Value",
-		".google.protobuf.Int64Value":  "runtime.Int64Value",
-		".google.protobuf.UInt64Value": "runtime.UInt64Value",
-		".google.protobuf.FloatValue":  "runtime.FloatValue",
-		".google.protobuf.DoubleValue": "runtime.DoubleValue",
-		".google.protobuf.BoolValue":   "runtime.BoolValue",
+		".google.protobuf.StringValue": "runtime.String",
```
Comment (mayankcpdixit) [inline protoc-gen-grpc-gateway/descriptor/types.go]: @johanbrandhorst seems it's still broken.

I try doing `go install` where my `*.pb.gw.go` files are.
Throws: `cannot assign string to protoReq.Domain (type *wrappers.StringValue) in multiple assignment`

Comment (mayankcpdixit) [reply protoc-gen-grpc-gateway/descriptor/types.go]: One way I could find is to update our `runtime/convert.go` to support `StringValue`.
```                                                                                                                                       
func StringValue(val string) (*wrappers.StringValue, error) {                                                                                
        return &wrappers.StringValue{Value: val}, nil                                                                                        
}
```

and keep the map here in `types.go` as:
```
".google.protobuf.BoolValue":   "runtime.BoolValue"
```

**Comment by johanbrandhorst** [comment]
Sigh it's clear to me that we need a little more time to work this out. I agree that your suggestion makes sense, can you submit a new PR? We need to add some examples in this area at the same time. I will revert my change to fix master.

**Comment by mayankcpdixit** [comment]
Sure.

**Pull request merged**